nginx/etc/nginx/nginx.conf.ctmpl

```diff
@@ -32,18 +40,23 @@ http {
         least_conn;
     }{{ end }}
 
-
     server {
         listen       80;
         server_name  _;
-
-        root /usr/share/nginx/html;
+        root         /usr/share/nginx/html;
 
         location /nginx-health {
             stub_status;
             allow 127.0.0.1;
             deny all;
+            access_log /var/log/nginx/access.log  main if=$isError;
```

Comment:
This [corresponding line](https://github.com/autopilotpattern/wordpress/blob/master/nginx/etc/nginx/nginx.conf.ctmpl#L20) has existed for sometime; I've not checked to see if this results in an on-disk file. The line I've added, in question, will result in LESS logging whatever the case may be.